sqlmock_test.go

```diff
@@ -354,6 +354,63 @@ func TestPreparedQueryExecutions(t *testing.T) {
 	}
 }
 
+func TestPreparedQueryMultipleExecutions(t *testing.T) {
+	t.Parallel()
+	db, mock, err := New()
+	if err != nil {
+		t.Errorf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+	defer db.Close()
+
+	mock.MatchExpectationsInOrder(false)
```

Comment:
well, in this case there is only one expectation, so there is always the one possible order. I agree in some tests, when there are more expectations and they are repeatable, then it requires to set it to unordered. But you should not turn both options on, by setting one. So that is fine in my opinion, it is better to have it explicit